src/index.js

```diff
@@ -49,11 +43,10 @@ function whichTripsToDisplay(tripStatus) {
 
 
 function clickLog() {
-    console.log("allDestinations", allDestinations)
-    // console.log("userData", userData)
-    // whichTripsToDisplay('approved');
-    // console.log(tripsToDisplay)
+    // console.log("allDestinations", allDestinations)
+    console.log("user", user)
+    console.log(user.trips)
 }
```

Comment:
Do we want console.logs in our main branch?

Reply:
No :(